support/api-testsupport/pom.xml

```diff
@@ -44,10 +49,6 @@
       <artifactId>wiremock</artifactId>
       <scope>compile</scope>
     </dependency>
-    <dependency>
-      <groupId>com.hotels.styx</groupId>
-      <artifactId>styx-common</artifactId>
-    </dependency>
-  </dependencies>
 
+  </dependencies>
```

Comment:
Is this change relevant? Looks like it could be backed off?

Let's try to minimise unnecessary changes. Later on if or when we need to consult the Git change log for the relevant changes, perhaps to find out what merge caused a bug, perhaps for another reason, irrelevant or stylistic changes are just a distraction. Therefore they should go into separate pull requests if possible.

Reply:
I moved it because I got briefly confused about the module relationships due to the `styx-common` dependency not being situated near `styx-api`, and wanting to prevent such confusion in the future. I don't think it merits an entirely separate PR though.